package.yaml

```diff
@@ -11,19 +11,31 @@ category:            Web
 
 dependencies:
   - base >= 4.7 && < 5
-  - http-dispatch
   - aeson
-  - text
   - bytestring
+  - http-client
+  - http-client-tls
+  - http-types
+  - network-uri
   - template-haskell
+  - text
+
+default-extensions:
+  - OverloadedStrings
+  - QuasiQuotes
+  - TemplateHaskell
 
 library:
-  source-dirs: lib
+  source-dirs: src
   exposed-modules:
-    - Network.Bugsnag.Types 
+    - Network.Bugsnag
+    - Network.Bugsnag.Client
+    - Network.Bugsnag.TH
+    - Network.Bugsnag.Types
 
 executables:
```

Comment:
This exe can be removed, since this is intended as a library. It may be worth writing a `Tutorial` module, but most of the stuff in the executable should really be used as test code I think